crates/router/src/connector/stax/transformers.rs

```diff
@@ -138,6 +163,28 @@ impl TryFrom<&types::TokenizationRouterData> for StaxTokenRequest {
                 };
                 Ok(Self::Card(stax_card_data))
             }
+            api_models::payments::PaymentMethodData::BankDebit(
+                api_models::payments::BankDebitData::AchBankDebit {
+                    billing_details,
+                    account_number,
+                    routing_number,
+                    bank_name,
+                    bank_type,
+                    bank_holder_type,
+                    ..
+                },
+            ) => {
+                let stax_bank_data = StaxBankTokenizeData {
+                    person_name: billing_details.name,
+                    bank_account: account_number,
+                    bank_routing: routing_number,
+                    bank_name: bank_name.unwrap_or(Secret::new("".to_string())),
+                    bank_type: bank_type.unwrap_or(Secret::new("".to_string())),
+                    bank_holder_type: bank_holder_type.unwrap_or(Secret::new("".to_string())),
```

Comment:
In case of None we should throw missing field error

Reply:
Done